src/services/NostrService.ts

```
@@ -85,37 +81,37 @@ export class NostrService {
-      let rating = {
-        total_reviews: 0,
-        total_rating: 0,
-        last_rating: 0,
-        max_rate: 5,
-        min_rate: 1
-      };
-      
+      let trades = 0;
+      let completion = 0;
+
       if (ratingTag && ratingTag[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)